lunatrace/bsl/hasura/migrations/lunatrace/1652400983913_add-pr-comment-disable/down.sql

```diff
@@ -1,10 +1,25 @@
-DROP TRIGGER add_project_settings ON public.projects;
-DROP TRIGGER add_organization_settings ON public.organizations;
 
-ALTER TABLE public.projects DROP CONSTRAINT project_settings_fkey;
-ALTER TABLE public.organizations DROP CONSTRAINT organization_settings_fkey;
+ALTER TABLE public.projects DROP COLUMN settings_id;
+ALTER TABLE public.projects ADD COLUMN settings_id uuid REFERENCES public.settings;
 
-ALTER TABLE public.settings DROP COLUMN pr_feedback_enabled;
-ALTER TABLE public.settings ADD COLUMN is_org_settings boolean;
+ALTER TABLE public.organizations DROP COLUMN settings_id;
+ALTER TABLE public.organizations ADD COLUMN settings_id uuid REFERENCES public.settings;
```

Comment:
Do we need to add tables to support organizational inheritance if it is not implemented? imo we should keep the schema as simple as possible until we have a plan for what we are going to do.